### ISSUE TITLE
setup.ini not working

### ISSUE THREAD

**Issue by yefoenix** [AntonyLeons/Ward#111]
I am useing the jar of v2.4.1，and made the file before starting.
```
[setup]
serverName = myserver
theme = dark
port = 8200
enableFog = true
```
but the app still exited with Error message:

> ***************************
> APPLICATION FAILED TO START
> ***************************
> 
> Description:
> 
> Web server failed to start. Port 4000 was already in use.
> 
> Action:
> 
> Identify and stop the process that's listening on port 4000 or configure this application to listen on another port.

**Comment by AntonyLeons** [comment]
you are correct, it needs port 4000 for launch then switches to the specified port, will try to fix asap.

**Comment by AntonyLeons** [comment]
fixed in https://github.com/AntonyLeons/ward-rs
